docs/modules/ROOT/pages/gsn-strategies.adoc

```diff
@@ -112,28 +118,26 @@ contract MyContract is GSNRecipient, GSNBouncerERC20Fee, MinterRole {
 }
 ----
 
-== Custom Bouncer
+== Custom strategies
+
+If the included strategies don't quite fit your business needs, you can also write your own one!  For example, your custom strategy could use a specified token to pay for relayed calls with a custom exchange rate to ether.  Alternatively you could issue users who subscribe to your dApp ERC721 tokens, and accounts holding the subscription token could use your contract for free as part of the subscription. There are lots of potential options!
 
-You can create your own Custom Bouncer!  For example, your Custom Bouncer could use a specified token to pay for relayed calls with a custom exchange rate to ether.  Alternatively you could issue users who subscribe to your dapp ERC721 tokens and accounts holding the subscription token could use your contract for free as part of the subscription.  There are lots of potential options for your Custom Bouncer.
+To write a custom strategy, simply inherit from `GSNRecipient` and implement the `acceptRelayedCall`, `_preRelayedCall` and `_postRelayedCall` functions.
 
-Your Custom Bouncer can inherit from `GSNBouncerBase` and must implement the `acceptRelayedCall` function.
+Your `acceptRelayedCall` implementation decides whether or not to accept the relayed call: return `_approveRelayedCall` to accept, and return `_rejectRelayedCall` with an error code to reject.
 
-Your `acceptRelayedCall` implementation decides whether or not to accept the relayed call.
-If your logic accepts the relayed call then you should return `_approveRelayedCall`.
-If your logic rejects the relayed call then you should return `_rejectRelayedCall` with an error code.
-See https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/GSN/bouncers/GSNBouncerSignature.sol[GSNBouncerSignature.sol] as an example implementation.
+`_preRelayedCall` and `_postRelayedCall` are not required for all GSN strategies (`GSNRecipientSignature` leaves them empty), but are useful when your strategy involves charging end users.
```

Comment:
"Not required" seems to imply you don't need to define them, but you do in order to explicitly specify that they are empty. Can you review this sentence with that in mind?